environments.ini

```diff
@@ -1654,6 +1654,24 @@ build_flags =
   '-DGateway_Name="OMG_ESP32_BLE"'
 custom_description = BLE gateway on the S3
 
+[env:esp32s3-m5stack-stamps3]
+platform = ${com.esp32_platform}
+board = esp32-s3-devkitc-1
+lib_deps =
+  ${com-esp32.lib_deps}
+  ${libraries.ble}
+  ${libraries.decoder}
+  ${libraries.adafruit_neopixel}
+build_flags =
+  ${com-esp32.build_flags}
```

Comment:
Could you also add TRIGGER_GPIO so that the board user settings could be erased with a long push:
https://github.com/1technophile/OpenMQTTGateway/blob/3b3fe3eabed675b51e96d75e5edabe897c35c7f4/environments.ini#L1802

Reply:
let me check

Reply:
the M5Stamp C3 and C3U have two buttons:
the C3U has a usable button on G9 : https://docs.m5stack.com/en/core/stamp_c3u
the C3 has a usable button on G3 : https://docs.m5stack.com/en/core/stamp_c3
the other button on C3 and C3U is hard reset 

the M5Stamp S3 **only** has a hard reset button on G0 : https://docs.m5stack.com/en/core/StampS3 

so - no, as no free button available and G0 didnt work for me as it immediately resets